### ISSUE TITLE
[Request] Option to mute darkmoon merry-go-round music

### ISSUE THREAD

**Issue by MirreNL** [leatrix/leatrix-sounds#3]
I have turned my ingame music off but the Darkmoon fair merry-go-round plays it's music anyway. Does not bother when you have one or two chars that visit it but when you have a lot of alts like me it really gets annoying.
Is ther any chance that you can add the merry-go=round music to the mutable sounds?

**Comment by leatrix** [comment]
Not tested but according to Leatrix Sounds, the IDs are 877255 and 772749.  So enter **MerryGoRound 877255, 772749** into 'Mute custom sounds' ('System') and click Mute.

**Comment by MirreNL** [comment]
Will it then be muted for my entire account?

**Comment by leatrix** [comment]
Yes.  Leatrix Plus saves your configuration settings (including your sound ID list) account-wide so they will be muted on every character.  You don't need to click the 'Mute' button when you login or anything.

**Comment by MirreNL** [comment]
That would be awesome. Tnx for your help! Saturday when the darkmoon starts I will see if it worked.

How do you determine what the sound ID is of a sound in WOW?

**Comment by leatrix** [comment]
Using Leatrix Sounds is a good start so if you enter **merrygoround** in LTS, you get the sound IDs above.  You can enter up to 10 keywords in any order in your search.  Any keywords prefixed with ! are excluded from search results.

If you enter something into LTS which produces a lot of results (such as searching for Darkmoon Faire), you can click the first result in the list then quickly traverse the listing by repeatedly pressing the S key.  With this method, you can listen to quite a lot of sounds in a short space of time to find the one you are looking for.

If you get stuck, you can always ask here.
